.claude/skills/implement/SKILL.md

```
@@ -227,17 +235,26 @@ After 3 failed attempts on the same issue, STOP and report to user:
 
 After all validations pass:
 
-```bash
-git add .
-git commit -m "feat: [milestone description]
+**Invoke the `commit` subagent:**
 
-🤖 Generated with Claude Code"
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)